torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -327,7 +326,7 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
         module_->register_module(field, submodule);
         auto v = module_->find_module(field);
         return std::make_shared<ModuleValue>(
-            m.graph()->insertGetAttr(self_, field), v, attr);
+            m.graph()->insertGetAttr(self_, field), v, result);
```

Comment:
why was this changed ?

Reply:
It was wrong before and added the Python value, not the created scriptmodule

Reply:
add test for this then ?